includes/classes/sync/push.php

```diff
@@ -305,31 +337,48 @@ public function loop_item_elements_and_map() {
 			if ( ! isset( $tab->fields ) || ! $tab->fields ) {
 				continue;
 			}
-
+			
 			// to handle fields in a tab
 			foreach ( $tab->fields as $element_index => $field ) {
 
 				// to handle components with multiple fields inside
 				$fields_data    = $field->component->fields ?? array( $field );
 				$component_uuid = 'component' === $field->field_type ? $field->uuid : '';
-
+				
 				$is_component_repeatable = false;
 				if($component_uuid) {
 					$metadata      = $field->metadata;
 					$is_component_repeatable = ( is_object( $metadata ) && isset( $metadata->repeatable ) ) ? $metadata->repeatable->isRepeatable : false;
 				}
 
+				$componentProcessed = false;
```

Comment:
Do we need this as the only other place I can see it get used is within a comment? Almost feels unnecessary to have the if check below for the `!$componentProcessed`

```php
if ( $component_uuid && !$componentProcessed) {
						$this->element->component_uuid = $component_uuid;
						$uuid = $component_uuid . '_component_' . $component_uuid;
						// $componentProcessed = true;
					}
```